### ISSUE TITLE
Split Button in Menu does not work for High Contrast White

### ISSUE THREAD

**Issue by SavannahBourgeois** [microsoft/fluentui#12446]
### Environment Information

- **Package version(s)**: all
- **Browser and OS versions**: Anaheim Edge Version 82.0.453.2 (Official build) dev (64-bit)

### Describe the issue:

When a split button in a context menu has keyboard focus in high contrast white, the text is not legible. Note that it works correctly for high contrast black.

### Please provide a reproduction of the issue in a codepen:
You can repro this on your context menu demo page using the split button in menu example. "ContextualMenu with checkable menu items and toggleable split button" https://developer.microsoft.com/en-us/fabric/#/controls/web/contextualmenu

#### Actual behavior:

Text is not legible, see screenshot below.
![image](https://user-images.githubusercontent.com/42548121/77700216-fa9f8500-6f70-11ea-8d71-ff9892bc20ff.png)

#### Expected behavior:

Text is legible

**Comment by aneeshack4** [comment]
@SavannahBourgeois Thanks for reporting this! I confirm repro - only on High Contrast White and not Black. I think this is related to #11742. Will investigate further to confirm that it's the same type of contrast bugs that the Edge team needs to address at the browser level.

**Comment by aneeshack4** [comment]
I'm trying to also make sure this is not a general SplitButton issue - it looks fine on its own so this could also be from the ContextualMenu styles (splitbutton doesn't seem to have high contrast [selector](https://github.com/microsoft/fluentui/blob/master/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts#L229) applied to it) from thought it's hard to confirm because when I installed Chromium, it overwrote the old Edge on my system so I need to figure out what it used to look like.

**Comment by SavannahBourgeois** [comment]
@aneeshack4  I tested in Edge and it also repros there so it does not seem to be a Chromium specific issue. Feel free to message me if you want more screenshots or want me to test anything :) If Edge is gone from your machine, you can also try testing in IE or use a tool like BrowserStack to test other browsers

**Comment by xugao** [comment]
looks like this is fixed, just tested in Edge Chromium.
